docs/reference/plugins.rst

```diff
@@ -37,6 +37,8 @@ Effects
    :members:
 .. autoclass:: FruityFastDist
    :members:
+.. autoclass:: FruityBloodOverdrive
```

Comment:
Place it in alphabetical order, below `FruityBalance`

Reply:
Fixed with cb69ddb20ff634f33a462778cd705c8e8c48f316